CONTRIBUTING.md

```diff
@@ -1,69 +1,70 @@
 
 ### Folder Structure
 A folder name is also Upper-Camel-Case with hypen between.
-Folder name should be a  Data structure Implementations, Algorithm Implementations, or a Problems name.
+Folder name should be a  Data structure Implementation, Algorithm Implementation, or a Problem name.
 E.g  folder Sort, should include all sorting algorithms implementation.
 E.g  folder Array should have problems solved with the array data-structure
 E.g  folder Two-Pointer should have have problems  solved with the two pointer algorithm
-E.g  folder Sliding-Window should have problems  solved with the sliding window algorithm.
+E.g  folder Sliding-Window should have problems solved with the sliding window algorithm.
+
+Create a new folder if one doesn't already exist for the Data-Structure, Algorithm, or problem you solved. Make a folder with the name of the Data-Structure or Algorithm. In the folder if their is no folder for the programming language create another folder with the programming language name. e.g Python then inside it create another folder for the Data-Structure e.g LinkedList, 
+inside LinkedList create source file for the Data-Structute e.g Js-Linked-List.js to write your solution.
+Add a ReadMe.md file with a brief description of the problem or algorithm, its complexity and runtime, and a link to an explanation blog post.
 
-If a folder does not exist for the Data-Structure you implemented, Algorithm or problem you solve create a new folder,
-Create a folder with the Data-Structure or Algorithm name, In the folder if their is no folder for the programming language create another folder with the Programming language name. e.g Python then iside it create another folder for the (Data-structute) e.g Linked-List, 
-Inside LinkedList create the another folder LinkedList create another file (Data-structute) e.g Js-Linked-List.js to write your solution.
-Add A Readme.md wo write a short problem or algorithm discription, Complexity/Runtime and a link to a blog post Explanation.
-Testing
-Be confident that your code works. When was the last time you committed a code change, your build failed, and half of your app stopped working? Mine was last week. Writing tests for our Algorithms will help us ensure the implementations are air tight even after multiple fixes and code changes.
+### Testing
+Be confident that your code works. When was the last time you committed a code change, your build failed and half of your app stopped working? Mine was last week. Writing tests for our Algorithms will help us ensure the implementations are air tight even after multiple fixes and code changes.
 
 It is advised that the algorithm file (module) does not contain any "live" code but rather just exports the function(s) needed to execute the algorithm.
 
 Please refrain from printing to the console in your implementation AND test code.
 
 
-Wriiten by @ismaelsadeeq 2022
+Written by @ismaelsadeeq 2022
+Edited by @SulaimanAminuBarkindo 2022
```

Comment:
This is not important here

Reply:
> This is not important here

Yes I agree, but what is your suggestion please. 
Should I remove both the Written by and Edited by or leave the Written by

Reply:
Edited by;

Reply:
Alright, I will remove the Edited by then

Reply:
Sure, thanks for your understanding

Reply:
I effect the changes requested. Please review. 
Thank you very much for your time

Reply:
Hello Man @Nakamcode  If the Review is completed we can resolve the conversation, merge the PR and close the issue.
Thanks

Reply:
yeah